lib/index.js

```diff
@@ -417,24 +419,29 @@ Utils.cloneTerse = function(input) {
  *  - -4d -6h will reduce the parsed date by 4 days and 6 hours
  *
  * @param  {String} field the date string to parse.
+ * @param  {String} locale the locale string OR options object.
  * @return {Object}       an object containing the parsed date, or the passed field if it was not a String.
  */
-Utils.parseDateTimeField = function(field) {
+Utils.parseDateTimeField = function(field, locale) {
```

Comment:
Since I wrote this comment, this feature has been implemented! So please ignore. :smile: